packages/ngtools/webpack/src/transformers/make_transform.ts

```diff
@@ -99,8 +99,9 @@ export function makeTransform(
 // If TS sees an empty decorator array, it will still emit a `__decorate` call.
 //    This seems to be a TS bug.
 function cleanupDecorators(node: ts.Node) {
-  if (node.decorators && node.decorators.length === 0) {
-    node.decorators = undefined;
+  if (node.decorators?.length === 0) {
+    // tslint:disable-next-line:no-any
+    (node as any).decorators = undefined;
```

Comment:
I'm curious if this workaround is still needed at all now.

Reply:
Can take a look when doing the upgrade.

Reply:
@clydin just checked this, and it seems that the issue is still present.

Expected
```
import { __decorate } from "tslib"; import { AnotherDecorator } from 'another-lib'; export class AppComponent { constructor() { this.title = 'app'; } onEscape() { console.log('run'); } } __decorate([ AnotherDecorator() ], AppComponent.prototype, "onEscape", null); AppComponent = __decorate([], AppComponent)
```
to equal
```
import { __decorate } from "tslib"; import { AnotherDecorator } from 'another-lib'; export class AppComponent { constructor() { this.title = 'app'; } onEscape() { console.log('run'); } } __decorate([ AnotherDecorator() ], AppComponent.prototype, "onEscape", null);'.
```